include/seqan/rna_io/rna_struct_file.h

```diff
@@ -0,0 +1,348 @@
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//     * Neither the name of Knut Reinert or the FU Berlin nor the names of
+//       its contributors may be used to endorse or promote products derived
+//       from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ==========================================================================
+// Author: Joerg Winkler <j.winkler@fu-berlin.de>
+// ==========================================================================
+// Class for reading/writing RNA structure files.
+// ==========================================================================
+
+#ifndef SEQAN_RNA_IO_RNA_STRUCT_FILE_H_
+#define SEQAN_RNA_IO_RNA_STRUCT_FILE_H_
+
+namespace seqan {
+
+// ==========================================================================
+// Tags, Classes, Enums
+// ==========================================================================
+
+// --------------------------------------------------------------------------
+// Tag RnaStruct
+// --------------------------------------------------------------------------
+
+/*!
+ * @tag FileFormats#RnaStruct
+ * @headerfile <seqan/rna_io.h>
+ * @brief General RNA structure file type. The concrete type is defined by the file name suffix.
+ * @signature typedef Tag<RnaStruct_> RnaStruct;
+ * @see FileFormats#Connect
+ * @see FileFormats#Stockholm
+ * @see FileFormats#DotBracket
+ * @see FileFormats#Vienna
+ * @see FileFormats#Bpseq
+ * @see FileFormats#Ebpseq
+ *
+ * This tag is used as TSpec template parameter in @link FormattedFile @endlink.
+ */
+struct RnaStruct_;
+typedef Tag<RnaStruct_> RnaStruct;
+
+typedef
+TagList<Connect,
```

Comment:
please indent the first TagList and put all others beneath on the same position.